internal/cmd/alpha/hana/check_test.go

```diff
@@ -31,9 +32,7 @@ func Test_checkHanaInstance(t *testing.T) {
 		err := checkHanaInstance(&config)
 		require.NotNil(t, err)
 		errMsg := err.String()
-		require.Contains(t, errMsg, "failed to get resource data")
```

Comment:
These missing cases will be cover in [this PR](https://github.com/kyma-project/cli/pull/2194/commits/27b11da6178f8d09c05dab9a89c51b1b1c15de03#diff-65f734d413e0fbdd8f790e58ab349542b1144e0b122ce3ecff21a9cdae74e7feR29)